### ISSUE TITLE
Thorium on musl linux systems

### ISSUE THREAD

**Issue by YusufKhalifadev** [Alex313031/thorium#616]
Thorium is not supported on musl linux distros like (alpine, voidlinux, etc) 

so it'd be perfect if there's a package for musl

**Comment by darkralts** [comment]
i agree and also another post here https://github.com/Alex313031/thorium/issues/312

**Comment by gz83** [comment]
Each thorium update requires uploading more than 40 files, which is a cumbersome process, and there may not be enough time to compile the musl version.
